Extensions/magiccards_popups.js

```diff
@@ -0,0 +1,93 @@
+//* TITLE magiccards.info Popups **//
+//* VERSION 0.1.4 **//
+//* DESCRIPTION    Creates visual popups for links to cards on https://magiccards.info/. **//
+//* DEVELOPER thebombzen **//
+//* FRAME false **//
+//* BETA true **//
+
+XKit.extensions.magiccards_popups = new Object({
+
+	running: false,
+
+	run: function() {
+		this.running = true;
+		if (XKit.interface.is_tumblr_page() === true) {
+			XKit.extensions.magiccards_popups.addPopupsIfNecessary();
+			XKit.tools.init_css("magiccards_popups");
+			XKit.post_listener.add("magiccards_popups", XKit.extensions.magiccards_popups.addPopupsIfNecessary);
+		}
+	},
+
+	destroy: function() {
+		this.running = false;
+		XKit.extensions.magiccards_popups.removePopups();
+		XKit.tools.remove_css("magiccards_popups");
+		XKit.post_listener.remove("magiccards_popups");
+	},
+
+	magiccards_regex: new RegExp("^https?://magiccards.info/([^/]+)/([^/]+)/([^.]+).html"),
```

Comment:
Wait, sorry. I'm testing the popups locally and the anchoring of this regex to the start of the string causes issues with Tumblr's tracking links of the form `http://t.umblr.com/redirect?z=https%3A%2F%2Fmagiccards.info%2Fod%2Fen%2F20.html&t=blahblahblahmorestuff`

Reply:
That's the purpose of lines 47-56.

Reply:
Oh that's strange. It didn't work the first time I tried it but it's working flawlessly now. Sorry for my confusion